sdk/lib/opentelemetry/sdk/trace/tracer.rb

```diff
@@ -16,14 +16,12 @@ class Tracer < OpenTelemetry::Trace::Tracer
         #
         # Returns a new {Tracer} instance.
         #
-        # @param [String] name Instrumentation package name
-        # @param [String] version Instrumentation package version
+        # @param [InstrumentationLibrary] resource Containing name and version arguments supplied to the TracerProvider
```

Comment:
This change is incorrect. We create the `InstrumentationLibrary` inside the method from the `name` and `version` params.

Reply:
The comment is incorrect, but the code looks fine.

Reply:
Yep, sorry - I should have been more specific: just the comment is incorrect.